query/control/controller.go

```diff
@@ -104,21 +105,21 @@ type Config struct {
 	// Less-scientifically, this was the only Config parameter other than ConcurrencyQuota to be typed as an int
 	// instead of an explicit int64. When ConcurrencyQuota changed to an int32, it felt like a decent idea for
 	// this to follow suit.
-	QueueSize int32
-
-	Logger *zap.Logger
-	// MetricLabelKeys is a list of labels to add to the metrics produced by the controller.
-	// The value for a given key will be read off the context.
-	// The context value must be a string or an implementation of the Stringer interface.
-	MetricLabelKeys []string
```

Comment:
`MetricLabelKeys` unused?

Reply:
It is used in 2.x for configuring scraping of the org name from the context. We don't do that in 1.x so I removed it.